pilot/docker/Dockerfile.proxy_debug

```diff
@@ -1,8 +1,8 @@
 # Version is the base image version from the TLD Makefile
 ARG BASE_VERSION=latest
 
-# The following section is used as base image if BASE_DISTRIBUTION=default
-FROM docker.io/istio/base:${BASE_VERSION} as default
+# The following section is used as base image if BASE_DISTRIBUTION=ubuntu
+FROM docker.io/istio/base:${BASE_VERSION} as ubuntu
```

Comment:
Can we use distroless debug image base here? No shell distroless image as proxy_debug doesn't make sense.

Reply:
This is actually running ubuntu still. We have a couple images that don't support distroless yet: https://istio.io/docs/ops/security/harden-docker-images/, this is one. I suspect the reasoning was that it doesn't make sense.

The distroless debug is great though, I didn't realize that exists. We can probably add support for distroless here and use the debug image

Reply:
Ah OK then that's fine, thanks for clarification.